Makefile

```diff
@@ -10,33 +13,25 @@ help:
 	sort | \
 	awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-
 clean:
 	find . -name "*.py[co]" -delete
 
-
 deps:
 	pip install pipenv --upgrade
 	pipenv install --dev
 
-
 format:
-	yapf -i --recursive $(SOURCE_CODE)
-	yapf -i --recursive tests
+	yapf -i --recursive $(SOURCE_CODE) $(TEST_CODE)
 
+lint:
+	yapf --recursive --diff $(SOURCE_CODE) $(TEST_CODE)
+	mypy $(SOURCE_CODE)
+	flake8 $(SOURCE_CODE)
 
 test:
-	yapf --recursive --diff $(SOURCE_CODE)
-	yapf --recursive --diff tests
-	mypy
-	flake8 ./nats/js/
 	pytest
 
-
 ci: deps
```

Comment:
No dependency on either the "lint" rule and the "test" one because of different corresponding commands.